command/state_rm.go

```diff
@@ -16,9 +16,8 @@ type StateRmCommand struct {
 func (c *StateRmCommand) Run(args []string) int {
 	args = c.Meta.process(args, true)
 
-	var backupPath string
```

Comment:
This is so bizarre cause this var was never actually used for anything. Its like I just forgot.